WordPress/src/main/java/org/wordpress/android/ui/posts/PostsListFragment.java

```diff
@@ -494,6 +505,36 @@ public void onPostsLoaded(int postCount) {
         } else if (postCount > 0) {
             hideEmptyView();
         }
+
+        // If the activity was given a target post, and this is the first time posts are loaded, scroll to that post
+        if (mTargetPost != null) {
+            if (mPostsListAdapter != null) {
+                final int position = mPostsListAdapter.getPositionForPost(mTargetPost);
+                if (position > -1) {
+                    RecyclerView.SmoothScroller smoothScroller = new LinearSmoothScroller(getActivity()) {
+                        private static final int SCROLL_OFFSET_DP = 23;
```

Comment:
Can we add a comment here explaining why we are moving this offset? maybe something like `// we're showing just a bit of the item above to hint the user they can navigate up` (feel free to re-write to something better worded than this)